enhanced-date.js

```diff
@@ -1 +1,116 @@
-// export this as a Node module using the code from Module_2
\ No newline at end of file
+'use strict';
+var dateObj = (function (){
```

Comment:
This is a node module, so the code should not be inside of the immediately invoked function expression. As it stands, your module won't export correctly.